frontend/src/components/forms/Signup.tsx

```diff
@@ -140,7 +142,10 @@ const Signup = ({
   };
 
   if (authenticatedUser) {
-    return <Redirect to={HOME_PAGE} />;
+    if (isVerified) {
```

Comment:
Is there a point of `isVerified`? It looks like it's always going to be false regardless